src/richText.js

```diff
@@ -731,6 +735,9 @@ define([
                    .replace(/<\/p>/ig, "\n")
                    .replace(/<br \/>/ig, "\n")
                    .replace(/(&nbsp;|\xa0|\u2005)/ig, " ")
+                   // While copying widgets with text, CKEditor adds these html elements
+                   .replace(/<span\b[^>]*?id="?cke_bm_\d+(\w)"?\b[^>]*?>.*?<\/span>/ig, "")
```

Comment:
Few questions
- Why group on `(\w)`? And is that intended to match a single word character or multiple?
- Is a quotation mark (") considered a word boundary? If not, would it be better to change the second `\b` to `\s`?
- Should the contents of the span be captured in a group and included in the replacement text?

Reply:
> Few questions
FYI - I adapted both the regex's from the CK code base itself so as to keep the match as close as possible.
Also see sample html for reference [here](https://github.com/dimagi/Vellum/pull/1081#issuecomment-1494450408)

> * Why group on `(\w)`? And is that intended to match a single word character or multiple?
My bad and good catch. Grouping is not needed at all and will be removing it. Intended to match single character as per the id format for bookmark.

> * Is a quotation mark (") considered a word boundary? If not, would it be better to change the second `\b` to `\s`?
Yes quotation mark is part of word boundary.

> * Should the contents of the span be captured in a group and included in the replacement text?
The bookmark span element has only fixed content: `&nbsp;`to support IE, which causes an additonal whitespace character in our plain text. This is not needed in replacement text.

Reply:
Looks like the extra group around `\w` has not been removed. Is that group in the original regex you copied from CK code? If yes, then fine to keep it so it matches as closely as possible.

Reply:
Sorry got missed and thanks for catching :). Yes the one I copied has it and is used for some processing. Since we don't need it, will get rid of the extra group.